src/components/Filter/filter-util.ts

```diff
@@ -120,6 +128,51 @@ const notequal = (items: any[], filterItem: FilterListInterface) => {
   }
 };
 
+const between = (items: any[], filterItem: FilterListInterface) => {
+  if (filterItem.value) {
+    return items.filter((obj) => {
+      const item = getFilteredPath(filterItem.path, obj);
+
+      return item >= filterItem.value.min && item <= filterItem.value.max;
```

Comment:
for easier readying can we change to this?
```suggestion
      return filterItem.value.min <= item && item <= filterItem.value.max;
```